src/pages/Login/Login.js

```diff
@@ -1,7 +1,105 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled, { css } from 'styled-components';
+import { FaPinterest } from 'react-icons/fa';
+import LoginModal from './LoginModal';
 
 const Login = () => {
-  return <div>Login</div>;
+  let [isLoginModalOpened, setLoginModal] = useState(false);
```

Comment:
state상태값과 상태를 변화시키는 함수명은 `set` 접두어 외에 동일한 단어로 작성하시는게 사용시에 헷갈리지 않을 것 같습니다. 
```suggestion
  let [isLoginModalOpened, setIsLoginModalOpened] = useState(false);
```

Reply:
네. 피드백 반영해서 수정했습니다